### DISCUSSION TITLE
NextAuth doesn't show basePath

### DISCUSSION THREAD

**Discussion by thebetterjort** [nextauthjs/next-auth#1519] [Help]
My setup works perfectly locally in docker, but when I try and deploy to a server behind a nginx proxy I'm not able to set the correct url for nextauth and it shows 404 error for 

/api/auth/session
/api/auth/_log

I've set this up in my .env.local:
NEXTAUTH_URL=https://example.com/fpd/api/auth

 next.config.js
module.exports = {
  basePath: '/fpd',

**Comment by balazsorban44** [comment]
could you see if that helps:

https://github.com/nextauthjs/next-auth/issues/689#issuecomment-699927237

**Reply by thebetterjort** [reply]
Yes, this is the same issue. Ty.
